### DISCUSSION TITLE
Multi-language support

### DISCUSSION THREAD

**Discussion by LidiaKovac** [AnasAito/SkillNER#42] [Q&A] [answered]
Hey guys! Do you have any plans in adding more languages to the module? If so, how can we help?

**Comment by Badr-MOUFAD** [comment, marked as answer]
Hello and thanks for your interest,
Actually, we do support translation in **SkillNer** even if we didn't mention it in the documentation.
However, the translation only goes forward (any language -> English) and not in both directions. Indeed, there is a small subtility in implementing it backward.

**Reply by LidiaKovac** [reply]
Amazing! Would that mean I can analyze italian job postings?

**Reply by Badr-MOUFAD** [reply]
Yes absolutely!
That being said, we will send a demo for your use case.

**Reply by AnasAito** [reply]
Hello @LidiaKovac  SKillNer take as an optional input a translation function, so he can map any language to English , we wrote you a small demo where we annotate an Italian job posting . 
Check this gist : https://gist.github.com/AnasAito/af2804196e6ac1169688862666a4eb4c

**Reply by LidiaKovac** [reply]
Thank you so much, I will make sure to put this to good use!
